solara/components/togglebuttons.py

```diff
@@ -133,15 +242,19 @@ def Page():
      * `children`: List of buttons to use as values.
      * `on_value`: Callback to call when the value changes.
      * `dense`: Whether to use a dense (smaller) style.
+     * `mandatory`: Whether selecting at least one element is mandatory.
      * `style`: CSS style to apply to the top level element.
      * `classes`: List of CSS classes to be applied to the top level element.
     """
     class_ = _combine_classes(classes)
     style_flat = solara.util._flatten_style(style)
-    reactive_value = solara.use_reactive(value, on_value)
+    # Similar approach to select.py
+    reactive_value = solara.use_reactive(value, on_value)  # type: ignore
     children = [solara.Button(label=str(value)) for value in values] + children
     allvalues = values + [_get_button_value(button) for button in children]
-    indices, set_indices = solara.use_state_or_update([allvalues.index(k) for k in reactive_value.value], key="index")
+    indices, set_indices = solara.use_state_or_update(
+        [allvalues.index(k) for k in reactive_value.value] if reactive_value.value is not None else None, key="index"
```

Comment:
```suggestion
        [allvalues.index(k) for k in reactive_value.value] if reactive_value.value is not None else [], key="index"
```
I think this is more consistent, instead of having both an empty list and None as valid values, we only use []